docs/source/sim_overview.rst

```diff
@@ -1,43 +1,41 @@
 Overview
 ========
 
+.. role:: strike
+
+Why run a simulation?
+---------------------
+
+Doing simulations can be a great way to assess how well ASReview performs for your particular purposes. The user can run simulations on previously labelled datasets to see - for example - how much time the user could have saved using ASReview, or how previous inclusion criteria could be improved in the future. The ASReview simulation mode works by sequentially going through the dataset like an ASReview user would, and uses the labels to learn from these inclusions and exclusions. In this way, the entire screening process is replicated.
 
 Doing the simulation
 --------------------
 
-Doing simulations can be a great way to assess how well *ASReview* performs for your particular
-needs.
-There are a few ways to perform simulations with the ASReview. For how to prepare your data,
-see the `dataset <datasets.html>`__ page.
-
-The first option is to use the simulation mode that is provided with the _ASReview_ package. It can
+You can use the simulation mode that is provided with the ASReview package. It can
```

Comment:
Why is the link to the dataset page removed?

Reply:
I don't know. I've added it again!